docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,692 @@
+:::
+
+The format for a GET request is the following:
+
+```js
+GET /{content-type}?_locale={locale-code}
+```
+
+Use `all` as a value for the locale code, as in `http://localhost:1337/restaurants?_locale=all`, to fetch entries for all locales that have been configured in the admin panel.
+
+If the `locale` parameter isn't defined, it will be set to the default locale. `en` is the default locale when i18n plugin is installed, so by default a GET request to `http://localhost:1337/restaurants` will return the same response as a request to `http://localhost:1337/restaurants?_locale=en`.
+.
+
+::: tip TIP
+Another locale can be [set as the default locale](/user-docs/latest/settings/managing-global-settings.md#adding-a-new-locale) in the admin panel.
+:::
+
+When the i18n plugin is installed, the response to requests includes fields that are specific to internationalization:
+
+- `locale` (string) is the locale code for the content entry
+- `localizations` (array) lists the existing localizations for this content entry; these localizations objects have 3 properties:
+  - `id` (number|string) is the id of the localized content entry
+  - `locale`(string) is the locale code for the localized content entry
+  - `published_at` (string) is the date and time when the localized content entry was [published](/developer-docs/latest/concepts/draft-and-publish.md#publishing-a-draft), in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format
+
+
+:::: tabs
+
+::: tab Request
+
+**Example request**
+
+```js
+GET http://localhost:1337/restaurants?_locale=fr
+```
+
+:::
+
+::: tab Response
+
+**Example response**
+
+```json
+[
+  {
+    "id": 4,
+    "name": "Can Alegria",
+    "description": "Can Alegria restaurant description in French",
+    "locale": "fr",
+    "localizations": [
+      {
+        "id": 3,
+        "locale": "en",
+        "published_at": "2021-04-07T10:10:31.949Z"
+      }
+    ]
+  },
+  {
+    "id": 8,
+    "name": "She's Cake",
+    "description": "She's Cake restaurant description in French",
+    "locale": "fr",
+    "localizations": []
+  }
+]
+```
+
+In the example response above:
+
+- restaurant with `"id": 4` is a French (`"locale": "fr"`) localization of the existing restaurant with `"id": 3` (created for the default `en` locale), as shown in the `localizations` object included in the response (see [creating a localization for an existing entry](#creating-a-localization-for-an-existing-entry)).
+- restaurant with `"id": "8"` was created from scratch using the API, passing the `locale: fr` in the request body (see [creating a new localized entry](#creating-a-new-localized-entry)).
+:::
+::::
+
+### Creating a new localized entry
+
+To create a localized entry from scratch, send a POST request to the Content API.
+
+If no locale has been passed in the request body, the entry is created using the default locale for the application:
+
+:::: tabs
+::: tab Request
+
+**Example request**
+
+```js
```

Comment:
Vuepress uses prismjs and this is their list of supported codes: https://prismjs.com/#supported-languages

Reply:
Thanks for the tip, Derrick! Honestly I was not too sure about what to use there when I wrote it, and re-used what we did for Content API.
Looking at the list you shared, `http` seems more appropriate.

I changed it [here](https://github.com/strapi/documentation/pull/224/commits/c2a40cb13854a920d4e58ff08e46aa931b36e5e5) and I'll create another PR for other similar code blocks in the repo.